### PR TITLE
Make tree master detail dialog styleable by adding css classes

### PR DESCRIPTION
**Pull request by lucas-koehler**
- Added style classes to dialog and its child elements
- Adapted tree renderer test cases to changes

### CODE REVIEW

File: src/renderers/additional/tree-renderer.ts [hunk 1]
```diff
@@ -139,15 +139,20 @@ export class TreeMasterDetailRenderer extends Renderer implements DataChangeList
...
+    title.classList.add('jsf-treeMasterDetail-dialog-title');
     this.dialog.appendChild(title);
     const dialogContent = document.createElement('div');
     dialogContent.classList.add('content');
+    dialogContent.classList.add('jsf-treeMasterDetail-dialog-content');
     this.dialog.appendChild(dialogContent);
     const dialogClose = document.createElement('button');
     dialogClose.innerText = 'Close';
     dialogClose.onclick = () => this.dialog.close();
+    dialogClose.classList.add('btn');
```
Comment (edgarmueller) [inline src/renderers/additional/tree-renderer.ts]: Why `btn`? Shouldn't this be `button`?

Comment (lucas-koehler) [inline src/renderers/additional/tree-renderer.ts]: This makes it compatible to materialize. However, that should be too specific for the general tree renderer. Should I change leave it, change it to `button` or remove it completely? I think having a general button class makes sense.

Comment (edgarmueller) [inline src/renderers/additional/tree-renderer.ts]: No, it's fine. It's just that we have to aggre on a style naming convention. I'd go with `button` in this case as and our tests already make use this.

File: src/renderers/additional/tree-renderer.ts [hunk 2]
```diff
@@ -139,15 +139,20 @@ export class TreeMasterDetailRenderer extends Renderer implements DataChangeList
...
+    title.classList.add('jsf-treeMasterDetail-dialog-title');
     this.dialog.appendChild(title);
     const dialogContent = document.createElement('div');
     dialogContent.classList.add('content');
+    dialogContent.classList.add('jsf-treeMasterDetail-dialog-content');
     this.dialog.appendChild(dialogContent);
     const dialogClose = document.createElement('button');
     dialogClose.innerText = 'Close';
     dialogClose.onclick = () => this.dialog.close();
+    dialogClose.classList.add('button');
```
Comment (edgarmueller) [inline src/renderers/additional/tree-renderer.ts]: Sorry, I've missed that the styling registry isn't utilized here. Use `JsonForms.stylingRegistry.addStyle` to apply the actual styles for the relevant HTML elements.

**Comment by edgarmueller** [comment]
Thank you!

**Pull request merged**